code/modules/admin/admin.dm

```diff
@@ -931,23 +949,23 @@ var/global/floorIsLava = 0
 
 /datum/admins/proc/adjump()
 	set category = "Server"
-	set desc="Toggle admin jumping"
+	set desc="Toggle Admin Jumping"
```

Comment:
I don't understand why you made the ones above lowercase and then made these ones uppercase.